__tests__/secrets.test.js

```diff
@@ -0,0 +1,62 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const UserService = require('../lib/services/UserService');
+
+const mockUser = {
+  firstName: 'Test',
+  lastName: 'Test Last Name',
+  email: 'test@test.com',
+  password: '12345',
+};
+
+describe('secrets routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  afterAll(() => {
+    pool.end();
+  });
+
+  it('can create a secret if signed in', async () => {
+    const agent = request.agent(app);
+    await UserService.create(mockUser);
+    const { email, password } = mockUser;
+    await agent.post('/api/v1/users/sessions').send({ email, password });
+
+    const expected = {
```

Comment:
We might also check that if a user isn't signed in the proper error message and status code are being sent